drivers/storage/portworx/manifest/manifest_test.go

```diff
@@ -35,6 +35,7 @@ func TestManifestWithNewerPortworxVersion(t *testing.T) {
 			PrometheusOperator:        "image/prometheus-operator:2.6.0",
 			PrometheusConfigMapReload: "image/configmap-reload:2.6.0",
 			PrometheusConfigReloader:  "image/prometheus-config-reloader:2.6.0",
+			Telemetry:                 defaultTelemetryImage,
```

Comment:
I would rather use something custom (`"image/telemetry:x.x.x"`) here instead of default, just to verify we don't always end up returning default image instead of what we receive from the install.portworx.com endpoint. Same for the tests below. 

To check we actually return default when not specified by install.portworx.com, add a default image check here - https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/manifest/manifest_test.go#L385

Reply:
Good idea. Done !